cardano-testnet/src/Testnet/Util/Runtime.hs

```diff
@@ -108,6 +123,17 @@ bftSprockets = fmap nodeSprocket . bftNodes
 poolSprockets :: TestnetRuntime -> [Sprocket]
 poolSprockets = fmap (nodeSprocket . poolRuntime) . poolNodes
 
+shelleyGenesis :: (H.MonadTest m, MonadIO m, HasCallStack) => TestnetRuntime -> m (ShelleyGenesis StandardCrypto)
+shelleyGenesis TestnetRuntime{shelleyGenesisFile} = withFrozenCallStack $
+  H.evalEither =<< H.evalIO (A.eitherDecodeFileStrict' shelleyGenesisFile)
+
+getStartTime :: (H.MonadTest m, MonadIO m, HasCallStack) => FilePath -> TestnetRuntime -> m UTCTime
+getStartTime tempRootPath TestnetRuntime{configurationFile} = withFrozenCallStack $ H.evalEither <=< H.evalIO . runExceptT $ do
+  A.Object cfg <- ExceptT $ A.eitherDecodeFileStrict' configurationFile
```

Comment:
Do we want to be more explicit about failures being thrown when the decoded JSON is not an object? cc: @newhoggy

Reply:
Good point!

Reply:
The only case when it would happen is when someone rewrites the configuration file with string, null, or a number. This would indicate that our test setup has failed, but it would also blow up earlier during testnet startup, because the configuration would be unreadable. I can add more verbose message if you feel that it would help here.

Reply:
What would the error message look like? Once it's obvious I think its ok.

Reply:
Not very obvious I'd say: something along the lines:
```
Pattern match failure in 'do' block at file:linenumber
````

Reply:
Then we should be explicit IMO.